layer/executables/packager.py

```diff
@@ -37,6 +40,10 @@ def package_function(
         # include cloudpickle itself in the executable
         _copy_cloudpickle_package(source)
 
+        if conda_env is not None:
+            conda_environment_file_path = source / "environment.yml"
```

Comment:
Is this conventional name for conda? If not, maybe rename to something more specific - `conda_env.yml`?

Reply:
Yes, it's called like that https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file